### DISCUSSION TITLE
Information on the Faces functionality

### DISCUSSION THREAD

**Discussion by lucasHabkost** [bpatrik/pigallery2#454] [Q&A]
Hi.
Starting by thanking you to put your time to this project. Really appreciated!

Can you please explain a bit further about the Faces fucntionality? 
I'm not expert in Photography so I dont know if I should expect that PiGallery gives functionality to tag a photo, or if that happens outside the app.
My point is I've loaded several thousand pictures and I dont see anything on Faces. 
Any help is highly appreciated.
Thanks!

**Comment by bpatrik** [comment]
Hi, 

thank you :)

The app does not do any ML, face detection and recognitions. You need to use an external software for that.
